Sources/BraintreeCore/Analytics/BTAnalyticsService.swift

```diff
@@ -6,16 +6,23 @@ class BTAnalyticsService: Equatable {
 
     /// The HTTP client for communication with the analytics service endpoint. Exposed for testing.
     var http: BTHTTP?
-    
+
     /// The FPTI URL to post all analytic events.
     static let url = URL(string: "https://api-m.paypal.com")!
 
+    // MARK: - Private Properties
+
     private let apiClient: BTAPIClient
+    private let timerInterval: Int
+    private let events = BTAnalyticsEventsStorage()
```

Comment:
If I toggle b/w features in the demo (Card, PP, Apple Pay), a new `BTAnalyticsEventsStorage()` is created each time (new empty array of events). Is this expected?

Reply:
You're right, not expected. Addressed in commit https://github.com/braintree/braintree_ios/pull/1295/commits/312a19d872ac31b93bef62256d8bd290659392ef by making events static. From there if you add the following under `await BTAnalyticsService.events.append(event)` and click through a number of flows before the 20s timer goes of each client will use same collection of events:
```swift
        print("START")
        await print("\(BTAnalyticsService.events.allValues) count: \(BTAnalyticsService.events.allValues.count)")
        print("END")
```